### ISSUE TITLE
Scale

### ISSUE THREAD

**Issue by fritz3721** [openstreetmap/openstreetmap-website#2219]
on the bottom of the osm-maps there is a scale showing for example the length of 3 km. I think it would make sense to not show only one step there, but to have a longer line ranging from one end of the screen to the other, so that distances can be easily measured. A ruler-tool would abviously be helpful as well.

**Comment by mwallnoefer** [comment]
I strongly agree with @fritz3721's requirement of introducing a _ruler_ respectively _measurement tool_ (https://wiki.openstreetmap.org/wiki/Front_Page_Design#What_design_elements_should_be_in_the_page?).

This defect forces me regularly to interrogate alternative map services in order to get the desired information. If the routing feature has been located in the nodes' context-menu for a long time it should not be hard to add measurements too.

**Comment by fazlerabbi37** [comment]
I am not sure if you guys got it correct but I think the scale in the bottom is not for measuring distance or as measurement tool. It's a scale that compares the map representation with actual world measure, meaning 1 cm in the map is equal to 1 km in real world. And in tradition map it is always in one corner, not in a whole page. See more at [Scale](https://en.wikipedia.org/wiki/Scale_(map)) wiki page.

**Comment by mwallnoefer** [comment]
@fazlerabbi37 Scale is okay, what I asked for was about a _measurement_ functionality as it is found in plenty of other map services (Google, Bing). Just open a node's context menu choose _Measure distance from_, open the target node and choose _Measure distance to_.
OpenStreetMap already provides a similar functionality, the routing planner, but sometimes the linear distance comes more handy.

**Comment by fazlerabbi37** [comment]
A measurement functionality would be nice to have! But unfortunately, the devs are pretty jammed up with existing tasks so don't hold your breath.

> OpenStreetMap already provides a similar functionality, the routing planner...

the route is provided by [OSRM](http://project-osrm.org/) which is not a part of core code functionality.
